hazelcast/src/main/java/com/hazelcast/quorum/QuorumException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.quorum;
+
+import com.hazelcast.transaction.TransactionException;
+
+/**
+ * An exception thrown when the cluster size is below the defined threshold.
+ */
+public class QuorumException extends TransactionException {
```

Comment:
taste of style.. I think it is OK and as far as I see QuorumException is pretty common.